.eslintrc.js

```diff
@@ -152,6 +152,13 @@ module.exports = {
         "no-undef": "off"
       }
     },
+    {
+      // There is a known issue with exporting async default functions
```

Comment:
Can you add a link to this issue please?

Reply:
Grabbing my links from last night, will update shortly...

Reply:
@jcoyne the link I included here is a closed issue, however they seem to go back and forth on this one. Perhaps there is a better syntax, but all suggested syntax ends up with a `fillInRequredFieldsForBibframeInstance is not a function` error. Open to any suggested better practices here.

Reply:
This is fine, thanks.